app/paysurvey/journey/ssj/SsjResponse.scala

```diff
@@ -1,12 +1,12 @@
 package paysurvey.journey.ssj
 
 import payapi.cardpaymentjourney.model.journey.Url
-import paysurvey.journey.JourneyId
+import paysurvey.journey.SurveyJourneyId
 import play.api.libs.json.{Format, Json}
 
 final case class SsjResponse
   (
-    journeyId: JourneyId,
+    journeyId: SurveyJourneyId,
     nextUrl:   Url
```

Comment:
Why do we need nextUrl here? I thoughts the clients will be building the url for survey with the SurveyJourneyId

Reply:
Its for the consumer to redirect too.This way if we make a change in the URL here we don't have to change it in the calling mircoservices